tests/unit/completion/test_completionwidget.py

```diff
@@ -145,13 +145,3 @@ def test_completion_item_focus(tree, count, expected, completionview):
         completionview.completion_item_focus(direction)
     idx = completionview.selectionModel().currentIndex()
     assert filtermodel.data(idx) == expected
-
-
-def test_completion_item_focus_no_model(completionview):
```

Comment:
Well, I did just find that this code handles things that don't have completions incorrectly: it uses the last completion opened rather than just not trying to complete. I don't think it's particularly useful to just check that `completion_item_focus` doesn't fail when the model is `None` if we don't know why there would be no model. Blindly passing when there is supposed to be a completion is still a bug; it's just less likely to be noticed. But I'll get back to this once I've corrected my code.